src/tests/regress/data/sql/strings.sql

```diff
@@ -648,13 +648,12 @@ select md5('12345678901234567890123456789012345678901234567890123456789012345678
 --
 -- encode/decode
 --
---@ SELECT encode('\x1234567890abcdef00', 'hex');
---@ SELECT decode('1234567890abcdef00', 'hex');
---@ SELECT encode(('\x' || repeat('1234567890abcdef0001', 7))::bytea, 'base64');
---@ SELECT decode(encode(('\x' || repeat('1234567890abcdef0001', 7))::bytea,
---@                      'base64'), 'base64');
```

Comment:
Keep them on 2 lines with the leading indentation spaces. In other words, only remove the 4 leading chars of `--@ ` and avoid any other changes, or the test output would not match.